config.yaml

```diff
@@ -72,6 +72,12 @@ options:
       The public-facing base URL that clients use to access this Homeserver.
       Defaults to https://<server_name>/. Only used if there is integration with
       SAML integrator charm.
+  ready_check:
```

Comment:
I think we should be careful about adding a config option with this name. To me this is a temporary workaround and something we'd want to remove at some point. Having a "experimental" prefix or something like that, as well as a note about it in the description would make this clearer.

Reply:
Good catch, renamed.